### DISCUSSION TITLE
Invitation: show support for Bash and Zsh completions in non-Linux installations of VS Code

### DISCUSSION THREAD

**Discussion by david-fong** [scop/bash-completion#943] [General]
VS Code ships Bash and Zsh completion scripts for its commandline interface for Linux platforms, and I'm pretty sure it would be little to no work for them to also ship them for non-linux platforms (Ex. Windows, where plenty of people use Git Bash, and macOS, where Zsh is the default shell). I've created a feature-request for this, and it needs to gain 20 upvotes within 60 days for it to be added to the VS Code team's backlog. Please do give it a thumbs up if you think it would be a useful change: https://github.com/microsoft/vscode/issues/180446.

**Comment by akinomyoga** [comment]
The remote ~~PR~~ Issue (**edit**: see https://github.com/scop/bash-completion/discussions/943#discussioncomment-9415657) seems to have already been closed.

**Reply by david-fong** [reply]
What PR exactly are you referring to?

**Reply by akinomyoga** [reply]
https://github.com/microsoft/vscode/issues/180446

Did you open another PR?

**Reply by david-fong** [reply]
It's not a PR. It was a feature-request issue ticket. It was closed due to not gaining enough community support votes in the allotted time window to get added to the backlog (which was my goal in raising this discussion).

**Reply by akinomyoga** [reply]
Ah, OK. Thank you for pointing it out. I've updated the comment.
